src/__tests__/medications/search/MedicationRequestTable.test.tsx

```diff
@@ -1,39 +1,49 @@
-import { render, screen, waitFor } from '@testing-library/react'
+import { render as rtlRender, screen, waitFor } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import { createMemoryHistory } from 'history'
-import React from 'react'
+import React, { ReactNode } from 'react'
 import { Router } from 'react-router-dom'
 
 import MedicationSearchRequest from '../../../medications/models/MedicationSearchRequest'
 import MedicationRequestTable from '../../../medications/search/MedicationRequestTable'
 import MedicationRepository from '../../../shared/db/MedicationRepository'
 import Medication from '../../../shared/model/Medication'
 
+type WrapperProps = {
+  // eslint-disable-next-line react/require-default-props
+  children?: ReactNode
+}
+
 describe('Medication Request Table', () => {
-  const setup = async (
+  const render = (
     givenSearchRequest: MedicationSearchRequest = { text: '', status: 'all' },
     givenMedications: Medication[] = [],
   ) => {
     jest.resetAllMocks()
     jest.spyOn(MedicationRepository, 'search').mockResolvedValue(givenMedications)
     const history = createMemoryHistory()
 
+    function Wrapper({ children }: WrapperProps) {
+      return <Router history={history}>{children}</Router>
+    }
+
+    const utils = rtlRender(<MedicationRequestTable searchRequest={givenSearchRequest} />, {
+      wrapper: Wrapper,
+    })
+
     return {
       history,
-      ...render(
-        <Router history={history}>
-          <MedicationRequestTable searchRequest={givenSearchRequest} />
-        </Router>,
-      ),
```

Comment:
this `render` pattern is preferred. Putting in a variable before spreading is fine though